k8s/src/main/scala/io/buoyant/k8s/ClientConfig.scala

```diff
@@ -27,6 +27,7 @@ trait ClientConfig {
   ) = {
     val setHost = new SetHostFilter(getHost, getPort)
     Http.client.withParams(Http.client.params ++ params)
+      .withHttpStats
```

Comment:
are we providing the client with a properly-scoped stats receiver? i assume so?

Reply:
yup.  final stat names look like `namer/#/io.l5d.k8s/client/http/status/4XX`